igor-web/src/main/java/com/netflix/spinnaker/igor/artifacts/ArtifactService.java

```diff
@@ -33,9 +33,10 @@
 
   /** Used to populate the manual trigger dropdown with options */
   @Nonnull
-  List<String> getArtifactVersions(@Nonnull String name, String releaseStatus);
+  List<String> getArtifactVersions(
+      @Nonnull String type, @Nonnull String name, List<String> releaseStatuses);
```

Comment:
This was previously limiting the ability to query artifacts with multiple statuses.